design model/components.model.json

```diff
@@ -0,0 +1,1170 @@
+                {
+                    "name": "ValueBrush",
+                    "valueType": "string",
+                    "value": ""
+                },
+                {
+                    "name": "TargetValueBrush",
+                    "valueType": "string",
+                    "value": ""
+                }
+            ],
+            "childComponents": [
+                {
+                    "id": "21-0",
+                    "name": "RangeBrush",
+                    "properties": [
+                        {
+                            "name": "Color",
+                            "valueType": "string",
+                            "value": ""
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "22",
+            "name": "LinearGauge",
+            "properties": [
+                {
+                    "name": "Value",
+                    "valueType": "number",
+                    "value": ""
+                },
+                {
+                    "name": "MinimumValue",
+                    "valueType": "number",
+                    "value": ""
+                },
+                {
+                    "name": "MaximumValue",
+                    "valueType": "number",
+                    "value": ""
+                },
+                {
+                    "name": "NeedleType",
+                    "valueType": "enum",
+                    "enumType": {
+                        "needle": "Needle",
+                        "triangle": "Triangle",
+                        "trapezoid": "Trapezoid"
+                    }
+                }
+            ],
+            "childComponents": [
+                {
+                    "id": "22-0",
+                    "name": "RangeBrush",
+                    "properties": [
+                        {
+                            "name": "Color",
+                            "valueType": "string",
+                            "value": ""
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "id": "23",
+            "name": "RadialGauge",
+            "properties": [
+                {
+                    "name": "Value",
+                    "valueType": "number",
+                    "value": ""
+                },
+                {
+                    "name": "MinimumValue",
+                    "valueType": "number",
+                    "value": ""
+                },
+                {
+                    "name": "MaximumValue",
+                    "valueType": "number",
+                    "value": ""
+                },
+                {
+                    "name": "NeedleType",
+                    "valueType": "enum",
+                    "enumType": {
+                        "needle": "Needle",
+                        "triangle": "Triangle",
+                        "trapezoid": "Trapezoid"
+                    }
```

Comment:
add needle brush, background color, major and minor ticks like for the linear gauge